rasa/core/brokers/file_producer.py

```diff
@@ -0,0 +1,55 @@
+import json
+import logging
+import typing
+from typing import Optional, Text, Dict
+
+from rasa.core.brokers.event_channel import EventChannel
```

Comment:
that's the only module which has `_producer` postfix in its name (PikaProducer or KafkaProducer don't). We should be consistent there

Reply:
this is to avoid a file named ``file.py`` while not adding an unnecessary ``_producer`` suffixes to the others. do you have another suggestion?

Reply:
Makes sense 👍 Nah, let it be, that's fine to me!